src/auth/auth-errors.ts

```diff
@@ -0,0 +1,2 @@
+export const sessionTimedOutErrorMessage =
+  'Session timed out (due to auth token refresh failure or expired refresh token)';
```

Comment:
```suggestion
  'Your session has expired.';
```

This is a user facing error message, so let's make it a little more user friendly. Users won't know about auth tokens or refresh tokens.